app.py

```
@@ -143,24 +167,30 @@ def on_exit():
 
 
 if __name__ == "__main__":
-    image_path = "med.gif"
-    app = RstEyeApp(image_path, fullscreen=True)
-    app.start()
+    try:
+        image_path = "med.gif"
+        logging.info("RstEyeApp is starting.")
+        app = RstEyeApp(image_path, fullscreen=True)
+        app.start()
+    except Exception as e:
```

Ruff

Do not catch blind exception: `Exception`

(BLE001)